src/routes/(shell)/identity/organizations/view/[id]/+page.svelte

```diff
@@ -38,28 +46,89 @@
 
 		Clients.identityClient(toastStore, at)
 			.apiV1OrganizationsOrganizationGet(parameters)
-			.then((v: Models.Organization) => (organization = v))
+			.then((v: Identity.Organization) => (organization = v))
 			.catch((e: Error) => Clients.error(e));
 
 		Clients.identityClient(toastStore, at)
 			.apiV1Oauth2providersGet()
-			.then((v: Models.Oauth2Providers) => (providers = v))
+			.then((v: Identity.Oauth2Providers) => (providers = v))
 			.catch((e: Error) => Clients.error(e));
+
+		Clients.identityClient(toastStore, at, [404])
+			.apiV1OrganizationsOrganizationOauth2providerGet(parameters)
+			.then((v: Identity.Oauth2Provider) => (provider = v))
+			.catch((e: Error) => {
+				// This is fine!
+				if (isErrorResponseError(e)) {
```

Comment:
Note: this shit show is mitigated by allowing named providers, and just returning an empty list, not a 404... while not needed now, it could facilitate organizations that are a federation of different domains!